src/modules/hip/kernel/warp_affine.hpp

```diff
@@ -0,0 +1,481 @@
+#include <hip/hip_runtime.h>
+#include "hip/rpp_hip_common.hpp"
+
+// -------------------- Set 0 - warp_affine device helpers --------------------
+
+__device__ void warp_affine_srclocs_hip_compute(float affineMatrixElement, float4 locSrcComponent_f4, d_float8 *locSrcPtr_f8)
+{
+    d_float8 increment_f8;
+    increment_f8.x.x = 0;
```

Comment:
use make_float4 and fill the increment_f8.x all at once.

Reply:
Done